modules/thermo-cycler/thermo-cycler-arduino/thermo-cycler-arduino.ino

```diff
@@ -696,6 +899,28 @@ void TC4_Handler()
   }
 }
 
+void check_saved_offsets()
+{ // If erroneous value or no offsets saved in EEPROM, then save the defaults to EEPROM
+  if (isnan(eeprom.get_offset_constant(OffsetConst::A)))
+  {
+    eeprom.set_offset_constant(OffsetConst::A, CONST_A_DEFAULT);
+  }
+  if (isnan(eeprom.get_offset_constant(OffsetConst::B)))
+  {
+    eeprom.set_offset_constant(OffsetConst::B, CONST_B_DEFAULT);
+  }
+  if (isnan(eeprom.get_offset_constant(OffsetConst::C)))
+  {
+    eeprom.set_offset_constant(OffsetConst::C, CONST_C_DEFAULT);
+  }
+}
+
+void update_offset_constants()
+{ // Should be called every time after updating EEPROM values
+  const_a = eeprom.get_offset_constant(OffsetConst::A);
```

Comment:
maybe explicit cast to void for the return here like `(void)eeprom.get_offset_constant(OffsetConst::A)`

Reply:
miread this, my bad